sidecar_acceptor_test.go

```diff
@@ -32,67 +31,46 @@ var (
 	}
 )
 
-func newTestDB(t *testing.T) (*clientdb.DB, func()) {
-	tempDir, err := ioutil.TempDir("", "client-db")
-	require.NoError(t, err)
-
-	db, err := clientdb.New(tempDir, clientdb.DBFilename)
-	if err != nil {
-		require.NoError(t, os.RemoveAll(tempDir))
-		t.Fatalf("unable to create new db: %v", err)
-	}
-
-	return db, func() {
-		require.NoError(t, db.Close())
-		require.NoError(t, os.RemoveAll(tempDir))
-	}
-}
-
 // TestRegisterSidecar makes sure that registering a sidecar ticket verifies the
 // offer signature contained within, adds the recipient node's information to
 // the ticket and stores it to the local database.
 func TestRegisterSidecar(t *testing.T) {
-	t.Parallel()
 
-	mockSigner := test.NewMockSigner()
-	mockWallet := test.NewMockWalletKit()
-	mockSigner.Signature = testOfferSig.Serialize()
-
-	acceptor := NewSidecarAcceptor(&SidecarAcceptorConfig{
-		SidecarDB:  nil,
-		AcctDB:     nil,
-		Signer:     mockSigner,
-		Wallet:     mockWallet,
-		NodePubKey: ourNodePubKey,
-		ClientCfg:  auctioneer.Config{},
-	})
+	emptySetter := func(
+		ticket *sidecar.Ticket, sc *test.MockSignerClient,
```

Comment:
I think we don't actually need the `ticket` in the setter at all.

Reply:
Totally right, I put it there at the beginning when I though that the mocks needed take thinks like the ID but they don't :)